README.md

```diff
@@ -43,6 +43,8 @@ API_BASE_URL=#{API_BASE_URL}
 
 ### Definition of Done
 
+- Recommended extensions have been installed if using VS Code
```

Comment:
As per my conversation with @adamcogan, @christianmorfordwaitessw, @pierssinclairssw  and @tkapa in the recent sprint. This will now be added as a task to the PR template